frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -29,6 +29,9 @@ export const normalizeClusterServiceVersions = (
         }`
       : operatorLogo;
 
+  const formatTileDescription = (desc) =>
+    `${desc.description}\n## Operator Description\n${desc.csv.spec.description}`;
```

Comment:
is there a possibility when `desc.csv.spec.description` would  get undefined

Reply:
Don't think so. The CSV data should contain a description.

Reply:
Just an FYI, we did upgrade TypeScript to 3.7 and there is something called Optional Chaining now https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/#optional-chaining - so checking for missing items is less of a concern.